Externals.cfg

```diff
@@ -1,7 +1,9 @@
 [ww3dev]
-tag = ww3_noresm0.0.9
+#tag = ww3_noresm0.0.9
+branch = feature/pass_10mwind_to_ww3
 protocol = git
-repo_url = https://github.com/NorESMhub/WW3.git
+#repo_url = https://github.com/NorESMhub/WW3.git
+repo_url = https://github.com/mvertens/WW3.git
```

Comment:
Is this necessary once the WW3 PR is merged?

Reply:
No. This is why this PR is a draft. As soon as the ww3dev PR gets merged and tagged I will update the Externals.cfg to the new ww3dev tag.